doc/source/cluster/kubernetes/getting-started/raycluster-quick-start.ipynb

```diff
@@ -0,0 +1,758 @@
+     "text": [
+      "Creating cluster \"kind\" ...\n",
+      " \u001b[32m✓\u001b[0m Ensuring node image (kindest/node:v1.26.0) 🖼\n",
+      " \u001b[32m✓\u001b[0m Preparing nodes 📦 7l\n",
+      " \u001b[32m✓\u001b[0m Writing configuration 📜7l\n",
+      " \u001b[32m✓\u001b[0m Starting control-plane 🕹️7l\n",
+      " \u001b[32m✓\u001b[0m Installing CNI 🔌7l\n",
+      " \u001b[32m✓\u001b[0m Installing StorageClass 💾7l\n",
+      "Set kubectl context to \"kind-kind\"\n",
+      "You can now use your cluster with:\n",
+      "\n",
+      "kubectl cluster-info --context kind-kind\n",
+      "\n",
+      "Not sure what to do next? 😅  Check out https://kind.sigs.k8s.io/docs/user/quick-start/\n"
+     ]
+    }
+   ],
+   "source": [
+    "kind create cluster --image=kindest/node:v1.26.0"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "9ef5b3ae-3404-4bb5-8894-f2719f39f4d6",
+   "metadata": {
+    "editable": true,
+    "slideshow": {
+     "slide_type": ""
+    },
+    "tags": []
+   },
+   "source": [
+    "(kuberay-operator-deploy)=\n",
+    "## Step 2: Deploy a KubeRay operator\n",
+    "\n",
+    "Deploy the KubeRay operator with the [Helm chart repository](https://github.com/ray-project/kuberay-helm) or Kustomize.\n",
+    "\n",
+    "### Method 1: Helm"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "648041da-62ac-45b3-bf9e-8bd6433f94d7",
+   "metadata": {
+    "editable": true,
+    "slideshow": {
+     "slide_type": ""
+    },
+    "tags": [
+     "nbval-ignore-output",
+     "remove-output"
+    ]
+   },
+   "outputs": [],
+   "source": [
+    "helm repo add kuberay https://ray-project.github.io/kuberay-helm/\n",
+    "helm repo update"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 42,
+   "id": "d48de329-1b64-4aa6-9ec5-33df23b4e2bb",
+   "metadata": {
+    "editable": true,
+    "slideshow": {
+     "slide_type": ""
+    },
+    "tags": [
+     "nbval-ignore-output",
+     "remove-output"
+    ]
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "NAME: kuberay-operator\n",
+      "LAST DEPLOYED: Tue Mar 11 21:50:15 2025\n",
+      "NAMESPACE: default\n",
+      "STATUS: deployed\n",
+      "REVISION: 1\n",
+      "TEST SUITE: None\n"
+     ]
+    }
+   ],
+   "source": [
+    "# Install both CRDs and KubeRay operator v1.3.0.\n",
+    "helm install kuberay-operator kuberay/kuberay-operator --version 1.3.0"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "4a4b8777-9fd4-4eea-a97d-0a40e4762864",
+   "metadata": {
+    "editable": true,
+    "slideshow": {
+     "slide_type": ""
+    },
+    "tags": []
+   },
+   "source": [
+    "### Method 2: Kustomize"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "03d55b9c-0479-4879-b9bd-e707d5a91391",
+   "metadata": {
+    "editable": true,
+    "slideshow": {
+     "slide_type": ""
+    },
+    "tags": [
+     "nbval-skip",
```

Comment:
Is it possible that we don't use `nbval-skip`? Maybe simply remove Kustomize from the doc? Or create another doc for Kustomize?

Reply:
I am okay to move Kustomize to another doc. @kevin85421 WDYT?

Reply:
BTW, moving Kustomize to another doc has another benefit: One can run all commands in the Jupyter notebook from the beginning instead of running only selected blocks.

Reply:
It appears that we’ll continue to keep Kustomize in the same document. Therefore, it’s fine to use `nbval-skip`.